### PR TITLE
update storage-js docs

### PR DESCRIPTION
**Pull request by inian**

### CODE REVIEW

File: spec/enrichments/tsdoc_v2/combined.json [hunk 1]
```diff
@@ -98093,7 +98093,7 @@
                           "kindString": "Parameter",
                           "flags": {},
                           "comment": {
-                            "shortText": "An array of files to be deleted, including the path and file name. For example [`folder/image.png`].\n"
+                            "shortText": "An array of files to be deleted, including the path and file name. For example [`'folder/image.png'`].\n"
```
Comment (dannykng) [inline spec/enrichments/tsdoc_v2/combined.json:98096]:
```suggestion
                            "shortText": "An array of files to delete, including the path and file name. For example [`'folder/image.png'`].\n"
```

File: spec/enrichments/tsdoc_v2/combined.json [hunk 2]
```diff
@@ -98272,7 +98272,7 @@
                           "kindString": "Parameter",
                           "flags": {},
                           "comment": {
-                            "shortText": "The relative file path. Should be of the format `folder/subfolder/filename.png`. The bucket must already exist before attempting to upload."
+                            "shortText": "The relative file path. Should be of the format `folder/subfolder/filename.png`. The bucket must already exist before attempting to update."
```
Comment (dannykng) [inline spec/enrichments/tsdoc_v2/combined.json:98275]: Is this "should" (recommended) or "must" (requirement)?

File: spec/enrichments/tsdoc_v2/combined.json [hunk 3]
```diff
@@ -98388,7 +98388,7 @@
                                     "isOptional": true
                                   },
                                   "comment": {
-                                    "shortText": "the `Cache-Control: max-age=<seconds>` seconds value."
+                                    "shortText": "The number of seconds the asset is cached in the browser and in the Supabase CDN. Defaults to 3600 seconds. This is set in the `Cache-Control: max-age=<seconds>` header."
```
Comment (dannykng) [inline spec/enrichments/tsdoc_v2/combined.json:98391]:
```suggestion
                                    "shortText": "The number of seconds the asset is cached in the browser and in the Supabase CDN. This is set in the `Cache-Control: max-age=<seconds>` header. Defaults to 3600 seconds."
```

File: spec/enrichments/tsdoc_v2/combined.json [hunk 4]
```diff
@@ -98434,7 +98434,7 @@
                                     "isOptional": true
                                   },
                                   "comment": {
-                                    "shortText": "whether to perform an upsert"
+                                    "shortText": "When upsert is set to true, the file will be overwritten if it exists. When set to false, an error will be thrown if the object already exists. Defaults to false."
```
Comment (dannykng) [inline spec/enrichments/tsdoc_v2/combined.json:98437]:
```suggestion
                                    "shortText": "When upsert is set to true, the file is overwritten if it exists. When set to false, an error is thrown if the object already exists. Defaults to false."
```

File: spec/enrichments/tsdoc_v2/combined.json [hunk 5]
```diff
@@ -98786,7 +98786,7 @@
                                     "isOptional": true
                                   },
                                   "comment": {
-                                    "shortText": "the `Cache-Control: max-age=<seconds>` seconds value."
+                                    "shortText": "The number of seconds the asset is cached in the browser and in the Supabase CDN. Defaults to 3600 seconds. This is set in the `Cache-Control: max-age=<seconds>` header."
```
Comment (dannykng) [inline spec/enrichments/tsdoc_v2/combined.json:98789]:
```suggestion
                                    "shortText": "The number of seconds the asset is cached in the browser and in the Supabase CDN. This is set in the `Cache-Control: max-age=<seconds>` header. Defaults to 3600 seconds."
```

File: spec/enrichments/tsdoc_v2/combined.json [hunk 6]
```diff
@@ -98832,7 +98832,7 @@
                                     "isOptional": true
                                   },
                                   "comment": {
-                                    "shortText": "whether to perform an upsert"
+                                    "shortText": "When upsert is set to true, the file will be overwritten if it exists. When set to false, an error will be thrown if the object already exists. Defaults to false."
```
Comment (dannykng) [inline spec/enrichments/tsdoc_v2/combined.json:98835]:
```suggestion
                                    "shortText": "When upsert is set to true, the file is overwritten if it exists. When set to false, an error is thrown if the object already exists. Defaults to false."
```

**Review by dannykng — COMMENTED** [review]
Left some comments and suggestions (hopefully in the right file).

File: spec/enrichments/tsdoc_v2/combined.json (hunk 2 above)
Comment (inian) [reply spec/enrichments/tsdoc_v2/combined.json:98275]: its a requirement Danny

**Comment by inian** [comment]
@dannykng thanks for the comments! made the changes [here](https://github.com/supabase/storage-js/commit/8481d27abf3b428b1199325fccf53d6279f1bf7b) and regenerated the docs

File: apps/reference/_supabase_js/generated/storage-from-remove.mdx [hunk 7]
```diff
@@ -34,7 +34,7 @@ const { data, error } = await supabase.storage
   </h4>
   <div class="method-list-item-description">
 
-An array of files to be deleted, including the path and file name. For example [`folder/image.png`].
+An array of files to be delete, including the path and file name. For example [`'folder/image.png'`].
```
Comment (dannykng) [inline apps/reference/_supabase_js/generated/storage-from-remove.mdx:37]:
```suggestion
An array of files to delete, including the path and file name. For example [`'folder/image.png'`].
```

File: spec/enrichments/tsdoc_v2/storage.json [hunk 8]
```diff
@@ -5934,7 +5934,7 @@
 											"kindString": "Parameter",
 											"flags": {},
 											"comment": {
-												"shortText": "An array of files to be deleted, including the path and file name. For example [`folder/image.png`].\n"
+												"shortText": "An array of files to be delete, including the path and file name. For example [`'folder/image.png'`].\n"
```
Comment (dannykng) [inline spec/enrichments/tsdoc_v2/storage.json:5937]:
```suggestion
												"shortText": "An array of files to delete, including the path and file name. For example [`'folder/image.png'`].\n"
```

**Review by dannykng — APPROVED** [review]
Couple minor typos but looks good otherwise!

**Pull request merged**